simp_le.py

```diff
@@ -257,19 +282,28 @@ class IOPlugin(object):
     """
     __metaclass__ = abc.ABCMeta
 
-    Data = collections.namedtuple('IOPluginData', 'account_key key cert chain')
+    Data = collections.namedtuple('IOPluginData',
+        'account_key account_reg key cert chain')
```

Comment:
This looks to be the source of the linter error.  It will usually take either of these styles:

```python
foo = bar(
   ...,
)
```

or:

```python
foo = bar(...,
          ...)
```